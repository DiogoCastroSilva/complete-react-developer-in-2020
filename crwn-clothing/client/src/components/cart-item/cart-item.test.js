import React from 'react';
import { shallow } from 'enzyme';

import CartItem from './cart-item.component';


describe('Testing CartItem component', () => {
    it('expects to render the CartItem component', () => {
        const mockProps = {
            imageUrl: 'https://store.storeimages.cdn-apple.com/4668/as-images.apple.com/is/iphone-12-pro-family-hero?wid=940&amp;hei=1112&amp;fmt=jpeg&amp;qlt=80&amp;op_usm=0.5,0.5&amp;.v=1604021663000',
            price: '1179â‚¬',
            name: 'Iphone 12 Pro',
            quantity: 1
        };

        expect(shallow(<CartItem item={mockProps} />)).toMatchSnapshot();
    });
});