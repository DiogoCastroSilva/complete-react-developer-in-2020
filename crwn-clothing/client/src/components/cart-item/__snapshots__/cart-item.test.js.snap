// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing CartItem component expects to render the CartItem component 1`] = `
<styled.div>
  <styled.img
    alt="item"
    src="https://store.storeimages.cdn-apple.com/4668/as-images.apple.com/is/iphone-12-pro-family-hero?wid=940&amp;hei=1112&amp;fmt=jpeg&amp;qlt=80&amp;op_usm=0.5,0.5&amp;.v=1604021663000"
  />
  <styled.div>
    <span>
      Iphone 12 Pro
    </span>
    <span>
      1
       x $
      1179â‚¬
    </span>
  </styled.div>
</styled.div>
`;
